css/index.css

```diff
@@ -205,11 +211,21 @@ header {
   border-bottom: 1px gray dashed;
   width: 100%;
 }
+@media (max-width: 500px) {
+  header {
+    display: flex;
+    flex-direction: column;
+    margin: 0 auto;
```

Comment:
I would also recommend trying out justify content in the future its really helpful